recipes/cpuinfo/all/conanfile.py

```diff
@@ -108,3 +108,6 @@ def package_info(self):
             self.cpp_info.components["cpuinfo"].requires = ["clog"]
         if self.settings.os in ["Linux", "FreeBSD"]:
             self.cpp_info.components["cpuinfo"].system_libs.append("pthread")
+
+        if self.settings.os == "Android":
+            self.cpp_info.components["cpuinfo"].system_libs.append("log")
```

Comment:
Thanks @elvisdukaj - this is correct as per: https://github.com/pytorch/cpuinfo/blob/main/CMakeLists.txt#L268-L270